ThingIFSDK/ThingIFSDK/HistoryStatesQuery.swift

```diff
@@ -9,7 +9,7 @@
 import Foundation
 
 /** Query to retrieve history of states. */
-open class HistoryStatesQuery: NSCoding {
```

Comment:
@yongpingchen 

We decided to remove `NSCoding` protocol because We assume that developers hardly save input data to storages.

The list to implement NSCoding or not is [here](https://docs.google.com/spreadsheets/d/1XZnFnt1glrHGAHKXzI3ske7BS1eUkD_l1i4Zag5CTUM/edit#gid=0)

The list is under review. After review, If we consider that this class should implement `NSCoding`. I add `NSCoding` to this class in other PR.